chapter_01.md

```diff
@@ -1,2 +1,550 @@
+
+As you might notice, Jenkins X is not only about Continuous Integration but also about Continuous Delivery. By covering Continuous Delivery, the pipeline doesn’t stop when these components are built. The pipeline is in charge of building, testing, and also deploying our artifacts into a live environment where they will run to serve our customers. The “continuous” part makes reference to the fact that you want to make sure deploying a new version of your service is frictionless and you will aim to deploy new versions in a short period of time. 
+
+In order to achieve Continuous Delivery, Jenkins X uses a set of conventions to enable developers to focus on building business value. These conventions are not exclusive to Jenkins X and are part of best practices gathered from different industries and practitioners. 
+
+One of these conventions is called “Trunk Based Development” which basically means that every change applied (merged) to the master branch will generate a new release of our artifacts. Most of the time, this is not comfortable for developers, as most of these practices are commonly defined in each company and they tend to vary quite a lot. The key motivation to use something like Trunk Based Development is to make sure that teams don’t need to spend time defining these practices. Projects like Jenkins X are the catalysts for thousands of community members, who are experts in the CI/CD space, which creates best practices and tools that apply them. When working with Trunk Based Development you are enabled to focus on writing code and, when your code is done and merged in master, a new release is created and deployed to some kind of staging environment for further validations. 
+
+I strongly recommend if you are starting a new project, to check the advantages of Trunk Based Development as well as the book Accelerate as it was used as the basis to create tools like Jenkins X. https://jenkins-x.io/about/overview/accelerate/
+
+At the end of the day, Jenkins X uses “One Repository / One Service” plus “Trunk Based Development” to take your service from source to a running instance inside a Kubernetes Cluster. 
+
+![chapter_01_05](images/chapter_01_05.png)
+
+For our example, the following links demonstrate all these concepts in action 
+Pipeline: https://github.com/salaboy/fmtok8s-email/blob/master/jenkins-x.yml 
+DockerFile: https://github.com/salaboy/fmtok8s-email/blob/master/Dockerfile
+Helm Charts: https://github.com/salaboy/fmtok8s-email/tree/master/charts/fmtok8s-email
+Continuous Releases: https://github.com/salaboy/fmtok8s-email/releases
+
+You will find the same setup for all the projects inside the Conference Site Demo. 
+
+
+#### Open APIs
```

Comment:
I suggest we rename this section, by linking the Open API to the DDD concept being presented. Maybe something like Defining APIs based on Bounded Contexts or something like this.

Reply:
Also, IMHO, we should highlight the concept and takeaways of the main idea, and then, create a subheader to show an actual implementation. 
E.g. for the subheading: "Implementing Open API in Spring Boot"
@otaviojava wdyt?